Documentation/Index.rst

```diff
@@ -0,0 +1,65 @@
+.. include:: Includes.txt
+
+.. _start:
+
+=============================================================
+EXT:tea
+=============================================================
+
+:Version:
+   |release|
+
+:Language:
+   en
+
+:Description:
+   This TYPO3 extension is an example of best practices in continuous integration and automated code checks, also
+   Extbase/Fluid-based extensions for TYPO3 CMS using PHPUnit, writing unit and functional tests for
+   extensions for TYPO3 CMS using PHPUnit.
+
+:Keywords:
+   tea, ci, continousintegration, gitlab, githubactions, actions, tests, functional, unit
+
+:Copyright:
+   2020 by TYPO3 Association
+
+:Authors:
+   * Oliver Klee
+
+:Email:
+   typo3-coding@oliverklee.de
+
+:License:
+   This extension is published under the
+   `GNU General Public License v2.0 <https://www.gnu.org/licenses/old-licenses/gpl-2.0.html>`__
+
+:Rendered:
+	|today|
+
+=====
+TYPO3
+=====
```

Comment:
According to https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/CheatSheet.html#rest-sphinx-cheat-sheet, there may only be one heading per file (marked with overlining and underlining), and the others need to be of the level header 1 etc. (i.e., without overlining). Ditto for the other non-title headers in this file.